### PR TITLE
OSDOCS-11706# Google Filestore auto delete resources

### PR DESCRIPTION
**Pull request by lpettyjo**
Reference this closed PR for previous conversations, etc. https://github.com/openshift/openshift-docs/pull/80454

Version(s): 4.17+

Issue: https://issues.redhat.com/browse/OSDOCS-11706

Link to docs preview: https://81387--ocpdocs-pr.netlify.app/openshift-enterprise/latest/storage/container_storage_interface/persistent-storage-csi-google-cloud-file.html#persistent-storage-csi-google-cloud-file-delete-instances_persistent-storage-csi-google-cloud-file

QE review:
- [x] QE has approved this change.

Additional information:

**PTAL: @tsmetana @gcharot No QE review required.**

### CODE REVIEW

File: modules/persistent-storage-csi-google-cloud-file-delete-instances.adoc [hunk 1]
```diff
@@ -3,22 +3,24 @@
...
 = Destroying clusters and GCP Filestore
 
-Typically, if you destroy a cluster, the {product-title} installer deletes all of the cloud resources that belong to that cluster. However, when a cluster is destroyed, Google Compute Platform (GCP) Filestore instances are not automatically deleted, so you must manually delete all persistent volume claims (PVCs) that use the Filestore storage class before destroying the cluster.
+Typically, if you destroy a cluster, the {product-title} installer deletes all of the cloud resources that belong to that cluster. However, due to the special nature of the Google Compute Platform (GCP) Filestore resources, the automated cleanup process might not remove all of them in some cases. 
+
+Therefore, Red Hat recommends that you verify that all cluster-owned Filestore resources are deleted by the uninstall process.
 
 .Procedure
-To delete all GCP Filestore PVCs:
+To ensure that all GCP Filestore PVCs have been deleted:
```
Comment (gcharot) [inline modules/persistent-storage-csi-google-cloud-file-delete-instances.adoc:14]: Stupid question, how can you use the oc command after the cluster has been destroyed?
cc @tsmetana

Comment (lpettyjo) [reply modules/persistent-storage-csi-google-cloud-file-delete-instances.adoc:14]: Good question!

Comment (tsmetana) [reply modules/persistent-storage-csi-google-cloud-file-delete-instances.adoc:14]: You can't. You have to use gcloud (either CLI or the web GUI), I'm afraid.

Comment (lpettyjo) [reply modules/persistent-storage-csi-google-cloud-file-delete-instances.adoc:14]: @tsmetana Ugh. I don't have way to emulate that to get the procedure. Who can help me out?

Comment (gcharot) [reply modules/persistent-storage-csi-google-cloud-file-delete-instances.adoc:14]: @tsmetana I think we're good with saying use google CLI/GUI but is there any label the customers can rely on to filter the filestore resources that were created by the cluster but not removed?

Comment (tsmetana) [reply modules/persistent-storage-csi-google-cloud-file-delete-instances.adoc:14]: Yes, there's the `kubernetes-io-cluster-${CLUSTER_ID}=owned` label -- the cluster ID is unique to the cluster, so there should not exist any filestores owned byt the deleted cluster after the uninstallation.

Comment (tsmetana) [reply modules/persistent-storage-csi-google-cloud-file-delete-instances.adoc:14]: > @tsmetana Ugh. I don't have way to emulate that to get the procedure. Who can help me out?

I agree with @gcharot, just mentioning to use the GCloud tools to verify the resources with the deleted cluster ID label are gone should be enough. I think in real-world scenarios the failure to wipe the filestores should be very rare.

Comment (lpettyjo) [reply modules/persistent-storage-csi-google-cloud-file-delete-instances.adoc:14]: Done.

**Comment by lpettyjo** [comment]
All comments incorporated. @tsmetana @gcharot  Need LGTM. Thanks!

**Pull request merged**